### ISSUE TITLE
Unable to resolve the name

### ISSUE THREAD

**Issue by prettyEric** [usnistgov/REFPROP-issues#530]
Hi,
As shown in the picture, I have installed python and ctREFPROP, but when I write [RP = py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary('C:\Program Files (x86)\REFPROP')] in MATLAB, the result is [Unable to resolve the name py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary]. How can I deal with that?

**Comment by ianhbell** [comment]
What version of MATLAB? There were some changes in very recent MATLAB that seem to have broken some python interfacing.

**Comment by prettyEric** [comment]
MATLAB 2021a

**Comment by ianhbell** [comment]
Can you do ``py.sys.path`` to see where your python is trying to load things from? Might give a hint.

**Comment by prettyEric** [comment]
I did [py.sys.path], and the result was ['', 'D:\\software\\Python\\python38.zip', 'D:\\software\\Python\\DLLs', 'D:\\software\\Python\\lib', 'D:\\software\\Python', 'C:\\Users\\prettyEric\\AppData\\Roaming\\Python\\Python38\\site-packages', 'D:\\software\\Python\\lib\\site-packages'].

**Comment by prettyEric** [comment]
The result was still [unable to resolve the name].

**Comment by prettyEric** [comment]
![屏幕截图 2022-05-24 200258](https://user-images.githubusercontent.com/101103538/170030163-702e6fab-2810-46d5-9dd2-1e44dbb398f4.png)

**Comment by ianhbell** [comment]
Maybe I am having a string escaping issue. What I am trying to do is: ``python -c "import ctREFPROP"``

**Comment by prettyEric** [comment]
![屏幕截图 2022-05-24 215406](https://user-images.githubusercontent.com/101103538/170052723-5d369de3-7e16-4f27-9030-c026af50d25a.png)

**Comment by ianhbell** [comment]
How did you get your python?

**Comment by prettyEric** [comment]
from the official website [https://www.python.org/downloads/], maybe I should reinstall it and try again?

**Comment by ianhbell** [comment]
I always use the python from conda, maybe that is part of the issue. I know that MATLAB does some funny things with the PATH, been burned by that before, but that doesn't explain why it wouldn't work.

At a command prompt, what do you get if you do:

``D:\software\python\python.exe -c "import ctREFPROP; print(ctREFPROP.__file__)"``

**Comment by prettyEric** [comment]
It did not work.
<img width="641" alt="屏幕截图 2022-05-24 233249" src="https://user-images.githubusercontent.com/101103538/170074975-1a2ebb7d-d0ba-486e-b8d5-1cd9c3afac2a.png">

**Comment by ianhbell** [comment]
I think your quotes are turned into unicode ones maybe? Can you type the double-quotes manually?

**Comment by prettyEric** [comment]
I tried, but the result was the same.

**Comment by ianhbell** [comment]
What if you do just D:\software\python\python.exe to get into a python shell, and after that do something like this:

```
(py39) C:\Users\ihb>python
Python 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:34:17) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import ctREFPROP
>>> ctREFPROP.__file__
'C:\\Users\\ihb\\AppData\\Local\\miniforge3\\envs\\py39\\lib\\site-packages\\ctREFPROP\\__init__.py'
```

**Comment by prettyEric** [comment]
still in the  command prompt?

**Comment by ianhbell** [comment]
Try what you did before

**Comment by prettyEric** [comment]
Still the same.
<img width="683" alt="屏幕截图 2022-05-25 002605" src="https://user-images.githubusercontent.com/101103538/170085674-1169f5e4-94c3-44fb-88c6-da63c8a86e3c.png">

**Comment by prettyEric** [comment]
Maybe I have to re-stall the python from conda.

**Comment by ianhbell** [comment]
Nah, I want you to still test the python in the command prompt.  We need to figure out why that doesn't work.

**Comment by prettyEric** [comment]
<img width="631" alt="屏幕截图 2022-05-25 003236" src="https://user-images.githubusercontent.com/101103538/170086811-d9ee52f2-4ea7-423d-b1d0-b3c025619b3c.png">

**Comment by ianhbell** [comment]
Did you copy the ``ctREFPROP.py`` script into your python directory at some point? It shouldn't find it at ``d:\software\python\ctREFPROP.py``

**Comment by ianhbell** [comment]
I tried on my side, worked just fine.

**Comment by ianhbell** [comment]
Can you try a whole lot of ``python -m pip uninstall ctREFPROP`` (until you get an error), followed by ``python -m pip install ctREFPROP`` and show me what that gives?

**Comment by ianhbell** [comment]
Don't do that.  Always (and only) install packages with pip.

**Comment by ianhbell** [comment]
Can you remove any other copies of ``ctREFPROP.py`` you have on your computer

**Comment by ianhbell** [comment]
Moral of the story: don't be copying things around, only install with pip.  Thanks for sticking with this as I had seen this issue before but have never succeeded in figuring out what was causing it.
